src/twisted/web/test/test_http.py

```diff
@@ -249,7 +298,9 @@ def test_noPipeliningApi(self):
         """
         b = StringTransport()
         a = http.HTTPChannel()
-        a.requestFactory = DummyNewHTTPHandler
+        requestFactory = _IDeprecatedHTTPChannelToRequestInterfaceProxy.wrap(
+            DummyHTTPHandler)
```

Comment:
It would probably be nice to hoist these type definitions.  It looks like many are repeated at least a couple times - `DummyHTTPHandler` many more than that.